x-pack/plugins/security_solution/scripts/endpoint/resolver_generator.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+require('../../../../../src/setup_node_env');
```

Comment:
@oatkiller  I'm a bit late - sorry 😄 
I was going to suggest that you also set permissions on this script to `755` and add the following shebang line to the top of file, so that we can just invoke it as a shell script:

```
#!/usr/bin/env node
```

I think you know this, but we (Endpoint Management) also use this script to load Endpoints 😬 

(FYI: https://github.com/paul-tavares/kibana/blob/9873df8ee0ee88584db70debc3316126ba3205af/x-pack/plugins/security_solution/scripts/endpoint/load_trusted_apps.js#L1 )

Reply:
Nah I didn't know that. I copied the pattern from somewhere else in Kibana.